layers/+tools/cfengine/packages.el

```diff
@@ -40,6 +40,6 @@
     :defer t))
 
 (defun cfengine/pre-init-org ()
-  (when (configuration-layer/package-used-p 'org)
+  (when (configuration-layer/layer-usedp 'org)
```

Comment:
`configuration-layer/package-usedp` is alias for `configuration-layer/package-used-p`. So I think that we should stick to the original function (left side).

Reply:
My bad. `layer` vs `package`